### ISSUE TITLE
Minor Bug: SendButton component receives an improper prop 'title'

### ISSUE THREAD

**Issue by fbartho** [GetStream/stream-chat-react-native#134]
**Describe the bug**
The SendButtonProps are documented (in TypeScript) as receiving a `title`, `editing`, `sendMessage()`

The the value in the title, appears to be related to a camera image picker however as seen in this screenshot:

<img width="641" alt="Screenshot of the title rendered into our button" src="https://user-images.githubusercontent.com/209712/73988679-3f575a00-48f8-11ea-9978-099c2d4417db.png">

**Dev environment info (please complete/provide the following information):**

- Package versions: 
```
"stream-chat": "1.3.x",
"stream-chat-react-native": "0.6.5",
```
- react-native version
- Detected on iOS Simulator Version 10.3 (SimulatorApp-880.5 CoreSimulator-681.17.2)

**To Reproduce**

Render the title parameter passed to the `SendButton` component that you pass into the `MessageInput` component

**Expected behavior**

Expected behavior -- remove the `title` prop or document what it's for / how its supposed to be used.

I would naively assume that the title should just be "Send" or "Send Message" (but localization is a concern).

**Screenshots**

Cropped Screenshot Provided Above, Full Screenshot here:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/209712/73988830-bd1b6580-48f8-11ea-9565-b72419307a7d.png">

**Comment by fbartho** [comment]
@vishalnarkhede Found the line of code here: https://github.com/GetStream/stream-chat-react-native/blob/ec248a0d2bffd7e96595c44b5a5f8769c7e6e7f9/src/components/MessageInput.js#L830

**Comment by vishalnarkhede** [comment]
Hey @fbartho thanks for brining this to our attention. I will fix this in next release :)

**Comment by vishalnarkhede** [comment]
@fbartho this has been fixed as part of `0.6.6`. now!
